### PR TITLE
games-simulation/simutrans: version bump

### PR DESCRIPTION
**Pull request by l29ah**

### CODE REVIEW

File: games-simulation/simutrans/simutrans-0.120.2.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+COLOUR_DEPTH=16
+OSTYPE=linux
+VERBOSE=1" > config.default || die
+
+	# make it look in the install location for the data
+	sed -i \
+		-e "s:argv\[0\]:\"/usr/share/${PN}/\":" \
+		simmain.cc || die
+
+	rm -f simutrans/{simutrans,*.txt}
```
Comment (monsieurp) [inline games-simulation/simutrans/simutrans-0.120.2.2.ebuild]: `rm -v .. || die`

Comment (chewi) [inline games-simulation/simutrans/simutrans-0.120.2.2.ebuild]: Let me guess, neither of you actually [tested that change](https://bugs.gentoo.org/show_bug.cgi?id=623176).

Comment (l29ah) [inline games-simulation/simutrans/simutrans-0.120.2.2.ebuild]: I have just copied the original ebuild for that matter and that worked, while the commit that eventually ended up in the repo does not.

Comment (chewi) [inline games-simulation/simutrans/simutrans-0.120.2.2.ebuild]: @monsieurp's suggested change was correct precisely for this reason. Never assume that the project's layout hasn't changed and have the ebuild abort if a command no longer makes sense. But of course, that requires you to test it. He should have tested it too.

**Pull request closed without merging**